ckine/figures/figure4.py

```diff
@@ -11,7 +11,7 @@
 
 ckineConc, cell_names_pstat, IL2_data, IL2_data2, IL15_data, IL15_data2 = import_pstat(combine_samples=False)
 _, _, IL2_data_avg, IL15_data_avg, _ = import_pstat(combine_samples=True)
-unkVec_2_15 = import_samples_2_15(N=1)  # use one rate
+unkVec_2_15_glob = import_samples_2_15(N=1)  # use one rate
```

Comment:
If we're only using one sample, where do the confidence intervals come from?

Reply:
This unkVec is only used for the first two plots (EC50 and Corr Coefs)- the dose response plot is handled by another function which pulls 100 samples independently of this.

Reply:
(plot_exp_pred is that function, and it's found in figure S5)